cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -941,12 +951,17 @@ func (csr *ClusterStateRegistry) GetIncorrectNodeGroupSize(nodeGroupName string)
 
 // GetUpcomingNodes returns how many new nodes will be added shortly to the node groups or should become ready soon.
 // The function may overestimate the number of nodes.
-func (csr *ClusterStateRegistry) GetUpcomingNodes() map[string]int {
+func (csr *ClusterStateRegistry) GetUpcomingNodes(time time.Time) map[string]int {
```

Comment:
Let's document what exactly the `time` parameter means here.

Reply:
Renamed it to match the rest, it's only there because of how weirdly CSR tracks backoffs.